apps/playground/src/app/components/popovers/dropdown/dropdown-routing.module.ts

```diff
@@ -27,6 +27,11 @@ const routes: ComponentRouteInfo[] = [
           import('./dropdown-example.component').then(
             (m) => m.DropdownExampleComponent,
           ),
+        data: {
```

Comment:
This and the change in `popovers.module.ts` fixes an error on the main component page where it was seeing two of the identical dropdown action buttons.